index.html

```diff
@@ -0,0 +1,58 @@
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<html>
+<!-- 
+
+Hello Camper!
+
+For now, the test suite only works in Chrome! Please read the README below in the JS Editor before beginning. Feel free to delete this message once you have read it. Good luck and Happy Coding! 
+
+- The freeCodeCamp Team 
+
+-->
+<head>
+    <link rel="stylesheet" type="text/css" href="styles.css">
```

Comment:
Fix indentation through the file. You can use tools like https://www.freeformatter.com/html-formatter.html but you should be very careful of indentation when you write. Take a few minutes to check how to configure VSCode to have tabs as 2 spaces (I believe it might be the default value, but I think it's best if you make it explicit). Usually in most IDE ctrl + , takes you to settings.